CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/hierarchical/ArrangeInHierarchyAction.java

```diff
@@ -55,6 +55,7 @@
 public class ArrangeInHierarchyAction extends AbstractAction {
 
     private final GraphNode context;
+    private static final String HELP_LOCATION = "au.gov.asd.tac.constellation.plugins.arrangements.hierarchical.HierarchicalAction";
```

Comment:
Perfect @aldebaran30701. Having "magic strings" taken out like this makes it way easier to know they exist so when we refactor code there is less chance they will be missed. Thank you!